assets/js/components/Link.js

```diff
@@ -35,13 +40,29 @@ function Link( {
 	caps,
 	danger,
 	disabled,
+	ariaLabel,
```

Comment:
Sorry @asvinb but I made a mistake here, and the prop for this should actually be `aria-label` as React doesn't treat these the same way as other html attributes (and actually warns about passing it as camel-case):

> Note that all `aria-*` HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML

I realize it's passed to the anchor this way but using different names for the prop could result in this one being overridden by a `aria-label` prop by the spread of `extraProps`.

```suggestion
	'aria-label': ariaLabel,
```
This would need to be updated elsewhere too but should be a straightforward rename.